requirements.txt

```diff
@@ -1,4 +1,4 @@
-nanopub==0.2.1
+git+https://github.com/fair-workflows/nanopub.git@fix_introduces_concept
```

Comment:
TODO: point to new nanopub release once https://github.com/fair-workflows/nanopub/pull/16 is merged

Reply:
PR #16 is now approved and merged :)